koku/masu/external/downloader/gcp/gcp_report_downloader.py

```diff
@@ -49,11 +49,23 @@ class GCPReportDownloaderError(Exception):
     """GCP Report Downloader error."""
 
 
+def get_ingress_manifest(manifest_id):
+    with ReportManifestDBAccessor() as manifest_accessor:
+        return manifest_accessor.get_manifest_by_id(manifest_id)
+
+
+def pd_read_csv(local_file_path):
+    try:
+        return pd.read_csv(local_file_path)
```

Comment:
This should be read in chunks to avoid hitting memory limits if the file is large.